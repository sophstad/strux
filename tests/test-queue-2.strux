int main() {
	Queue<num> q = new Queue<num>();
	num x = 3.0;
	q.offer(x);
	return 0;
}