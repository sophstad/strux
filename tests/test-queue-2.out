6.000000
50
mil
0
